VideoWeb/VideoWeb/Health/HealthCheckExtensions.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using VideoWeb.Common.Configuration;
+
+namespace VideoWeb.Health;
+
+[ExcludeFromCodeCoverage]
+public static class HealthCheckExtensions
+{
+    public static IServiceCollection AddVhHealthChecks(this IServiceCollection services)
+    {
+        var container = services.BuildServiceProvider();
+        var connectionStrings = container.GetService<ConnectionStrings>();
+        var servicesConfiguration = container.GetService<IOptions<HearingServicesConfiguration>>().Value;
+        services.AddHealthChecks()
+            .AddCheck("self", () => HealthCheckResult.Healthy())
+            .AddRedis(connectionStrings.RedisCache, tags: new[] {"startup", "readiness"})
+                // .AddUrlGroup(
+                // need a private endpoint o check signalr health
+                // new Uri(
+                //     new Uri(new ServiceEndpoint(connectionStrings.SignalR).Endpoint),
+                //     "/api/v1/health"),
+                // name: "SignalR Azure Hub",
+                // failureStatus: HealthStatus.Unhealthy,
+                // tags: new[] {"startup", "readiness"})
```

Comment:
do we need the commented out bit?

Reply:
ah this was a reminder to have a chat with Ben to see if there's a recommended approach we can take or use the current exposed endpoint